content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Burn the bootloader on SAMD boards
+description: Learn how to burn the bootloader on SAMD boards.
+difficulty: Advanced
+featuredImage: 'clip'
+---
+
+In this tutorial, we will learn how to burn the bootloader on boards that use the ATSAMD21G18 microcontroller using another Arduino board. The bootloader is a small piece of software that allows uploading sketches onto the Arduino board. It comes preprogrammed on the microcontrollers on Arduino boards. Whether the bootloader has been corrupted or intentionally has been removed, it can be restored by burning (also called, flashing or programming) a new bootloader to the board.
+
+Below you will find two different approaches to burning the bootloader:
+- The first approach uses an SD card and should be used when wanting to flash a different board than the ones bundled with the "flash_MKR_bootloaders" (check out the [Github repository](https://github.com/adafruit/Adafruit_DAP) for more information).
+
+- The second approach does not require a SD card and should be used by users who want the easiest solution.
```

Comment:
```suggestion
- [The second approach](#no-sd-card) does not require an SD card and should be used by users who want the easiest solution.
```

- Link to the relevant section of the page to make it easier for the reader to find it.
- Grammar nitpick.
  "SD" is pronounced "ess dee", and the choice of "a" vs "an" is based on pronunciation rather than spelling.